src/30_Advanced/SeatMap.html

```diff
@@ -41,36 +43,42 @@
         fill: var(--abe-light-pink);
         pointer-events: none;
       }
-      .selected {
-        fill: var(--abe-pink);
-      }
       .disabled {
         pointer-events: none;
         cursor: default;
         opacity: 0.5;
         filter: alpha(opacity=50);
       }
-      .seat-map-svg {
-        box-sizing: border-box;
-        padding-top: 30px;
-      }
       /* `amp-pan-zoom` has a default display: flex to center positioning the child. 
       In this sample we are going to use a seatmap placeholder and for correctly positioning it, we are overwriting the
       `amp-pan-zoom` display to `initial` */
-      #pan-zoom {
+      .seatmap {
         display: initial;
```

Comment:
Just noticed this, why do we need this? Shouldn't this be the default?

Reply:
It's explained above, display initial is not the default css for amp-pan-zoom.

Reply:
Sorry, should've read that